index.html

```diff
@@ -112,11 +115,9 @@ <h2 class="works-title">My Recent Works</h2>
           <div class="line"></div>
         </div>
         <ul class="frames">
-          <li class="frame frame-one">
-            <h3 class="title-post">Profesional Art Printing Data</h3>
-            <p class="title-post-text">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard
+          <li id="frameOne" class="frame frame-one" >
+            <h3 id="titlePost" class="title-post"></h3>
+            <p id="titlePostText" class="title-post-text">
             </p>
```

Comment:
- [ ] It is important to note that the projects section is supposed to be rendered dynamically. Therefore, all the codes in the projects/works section in the HTML file should be removed. Please remove them here in your HTML file 👍 

**Project requirements**
![image](https://user-images.githubusercontent.com/90222110/189173026-181dd41d-e23a-4f49-b6bd-01ded2da8312.png)